### PR TITLE
Localization for release 2.30

### PR DESCRIPTION
**Pull request by jenskasemets**

### CODE REVIEW

File: Resources/de.lproj/Push.strings [hunk 1]
```diff
@@ -28,6 +28,14 @@
 "push.notification.add.message.oneonone" = "%1$@: %2$@";
 "push.notification.add.message.oneonone.nousername" = "Neue Nachricht: %1$@";
 
+"push.notification.add.unknown.group" = "%1$@ sent a message in %2$@";
+"push.notification.add.unknown.group.nousername" = "New message in %1$@";
```
Comment (mikeger) [inline Resources/de.lproj/Push.strings]: No german localization?

Comment (daehn) [inline Resources/de.lproj/Push.strings]: Something went wrong here I think

File: Resources/de.lproj/Push.strings [hunk 2]
```diff
@@ -28,6 +28,14 @@
...
+"push.notification.add.unknown.group.nousername.noconversationname" = "Neue Nachricht";
+"push.notification.add.unknown.group.noconversationname" = "%1$@ sent a message";
+
+"push.notification.add.unknown.oneonone" = "%1$@ sent a message";
+"push.notification.add.unknown.oneonone.nousername" = "Neue Nachricht";
```
Comment (mikeger) [inline Resources/de.lproj/Push.strings]: No german localization?

File: Resources/de.lproj/Push.strings [hunk 3]
```diff
@@ -28,6 +28,14 @@
 "push.notification.add.message.oneonone" = "%1$@: %2$@";
 "push.notification.add.message.oneonone.nousername" = "Neue Nachricht: %1$@";
 
+"push.notification.add.unknown.group" = "%1$@ sent a message in %2$@";
+"push.notification.add.unknown.group.nousername" = "New message in %1$@";
+"push.notification.add.unknown.group.nousername.noconversationname" = "Neue Nachricht";
+"push.notification.add.unknown.group.noconversationname" = "%1$@ sent a message";
+
+"push.notification.add.unknown.oneonone" = "%1$@ sent a message";
```
Comment (mikeger) [inline Resources/de.lproj/Push.strings]: No german localization?

**Comment by jenskasemets** [comment]
Looks like there was some delay between approving these translations on Crowdin and them being available through their API, running job few minutes later fetched them nicely.

**Pull request merged**